### DISCUSSION TITLE
잠시 후 시도하세요 주기 시간 및 사용법

### DISCUSSION THREAD

**Discussion by HANUL2MERRY** [changdoc/naver-vaccine-macro#183] [Q&A]
안녕하세요 선생님

실수로 복수의 병원 돌리다가, 잠시후 시도 뜨길래 한시간 정도 시도 뒤에

지도 메인에서 새로고침 하기 눌렀다가
확장앱에 가이드대로 등록한 지역에 병원 리스트에서 하나 클릭해서 신청페이지 열어봤더니
잠시 시도중이 계속 나오더군요.

혹시 오늘부터 주기시간이 변경된걸까요? ㅠㅜ

그리고 사용법이 조금 혼동이 되는데요.
graphql result 등록을 해두고

세부 병원페이지에서만 돌리면 되는거예요?
아니면 전체시도에서도 가만히 둬도 상관없는건지 혹은 둘다 하는것을 권장하시는지 궁금합니다.

**Comment by changdoc** [comment]
주기 시간이 변경됐다는건 따로 파악하지 못했네요.

병원 예약 신청 페이지에서 돌리는 것과 지도에서 돌리는 것과의 병행은 본인 선택인 듯 합니다.

**Reply by HANUL2MERRY** [reply]
감사합니다.
